### ISSUE TITLE
aria-hidden explicitly sets display to none

### ISSUE THREAD

**Issue by libremente** [italia/bootstrap-italia#419]
## Comportamento atteso

The `aria-hidden` behaviour should be left to the HTML5 interpreter and not forced by explicit CSS rules. 

## Comportamento attuale

Due to this code:
```css
div[aria-hidden='true'] {
  display: none;
}
```
to be found in `src/scss/custom/_timepicker.scss:10`, the behaviour explained above is forced via CSS.

## Possibili soluzioni

Remove the three lines indicated above.
